staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/initializer/initializer_test.go

```diff
@@ -26,9 +26,9 @@ import (
 
 type doNothingAdmission struct{}
 
-func (doNothingAdmission) Admit(a admission.Attributes) error { return nil }
-func (doNothingAdmission) Handles(o admission.Operation) bool { return false }
-func (doNothingAdmission) Validate() error                    { return nil }
+func (doNothingAdmission) Admit(a admission.Attributes) error    { return nil }
+func (doNothingAdmission) Handles(o admission.Operation) bool    { return false }
+func (doNothingAdmission) Validate(a admission.Attributes) error { return nil }
```

Comment:
I think this function should be `ValidateInitialization` instead of `Validate`. Right?

Reply:
NewPluginInitializer only requires a admission.Interface:
https://github.com/kubernetes/kubernetes/blob/eb3196b1b46f69cb8257692017f741248438d4d9/staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/initializer/initializer.go#L66

So only function Handles() is required. What about delete Validate() and Admit() ?

Reply:
> What about delete Validate() and Admit() ?

Go ahead. Let the CI tell us the answer.

Reply:
Would drop `Admit` and `Validate`. Not even `ValidateInitialization` is needed.

Reply:
Done.